core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -292,7 +292,7 @@ class SparkContext(config: SparkConf) extends Logging {
   private[spark] val executorEnvs = HashMap[String, String]()
 
   // Set SPARK_USER for user who is running SparkContext.
-  val sparkUser = Utils.getCurrentUserName()
+  val sparkUser: String = Utils.getCurrentUserName()
```

Comment:
Although pretty harmless, I don't know if this type adds clarity, and we don't specify it by default